app/views/coronavirus_form/confirmation.html.erb

```diff
@@ -2,3 +2,6 @@
 <%= render "govuk_publishing_components/components/panel", {
   title: t("coronavirus_form.confirmation.title")
 } %>
+<%= render "govuk_publishing_components/components/lead_paragraph", {
```

Comment:
We don't need a component to render this `<%= sanitize(t('coronavirus_form.confirmation.description')) %>` should do just fine